CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ Changelog for `ta4j`, roughly following [keepachangelog.com](http://keepachangel
 - :tada: **Enhancement** Added ADX indicator based strategy to ta4j-examples  
 - :tada: **Enhancement** TrailingStopLossRule: added possibility of calculations of TrailingStopLossRule also for open, high, low price. Added getter 
 for currentStopLossLimitActivation
+- :tada: **Enhancement** Add constructors with parameters to allow custom implementation of ReportGenerators
```

Comment:
I think there is no information where enhancement was added. It could be ".. to allow custom implementation of ReportGenerators **in BacktestExecutor**".

Code changes are fine. Useful improvement!

Reply:
Agreed! Done